src/Silk.Core/Main.cs

```diff
@@ -64,9 +80,23 @@ private async Task InitializeClientExtensions()
             await ShardClient.UseInteractivityAsync(DiscordConfigurations.Interactivity);
         }
 
+        private Task InitializeSlashCommandsAsync()
+        {
+            _logger.LogInformation("Initializing Slash-Commands");
+            var sc = ShardClient.ShardClients[0].UseSlashCommands(DiscordConfigurations.SlashCommands);
+            sc.SlashCommandErrored += _slashExceptionHandler.Handle;
+            
+            sc.RegisterCommands<RemindCommands>(847615746745958520);
```

Comment:
We'll probably want at some point to think of how to make this a setting or someway for the user to change this because this will throw an exception for anyone who doesn't have access to those Guilds (`GuildId`'s) when registering Slash Commands

Reply:
Oops thanks for reminding me I forgot to remove this so it registers as global commands.